### PR TITLE
Improving entity CRUD operations and adding new methods

### PR DESCRIPTION
**Pull request by dwendland**
* Allow to set entity IDs for entity CRUD
* Add support for subscriptions and notifications
* Add support for Sidecar-Proxy [endpoint-configuration-service](https://github.com/FIWARE/endpoint-auth-service/tree/main/src/endpoint-configuration-service)

### CODE REVIEW

File: i4trust_service.py [hunk 1]
```diff
@@ -88,119 +87,260 @@ def on_post_product_spec_validation(self, provider, asset):
...
+                                            "Notification Identifiers",
+                                            "IDs of corresponding subscriptions",
+                                            asset.meta_info['notification_ids'])
+
+            # Add Sidecar-Proxy endpoint config service
+            if asset.meta_info['sidecar_endpoint_config_allowed']:
+                self._append_string_charact(charact,
+                                            "Sidecar-Proxy endpoint configuration service",
+                                            "Access to sidecar-proxy endpoint configuration service allowed",
+                                            "True") # TODO: Can one use boolean instead?
```
Comment (dwendland) [inline i4trust_service.py:342]: @fdelavega Is it possible to add characteristics of type boolean instead of string?

Comment (fdelavega) [reply i4trust_service.py:342]: Yes, it is possible

**Review by fdelavega — APPROVED** [review]
I looks good to me

File: i4trust_service.py (hunk 1 above)
Comment (dwendland) [reply i4trust_service.py:342]: What is the name of the function then?

**Pull request merged**